walkthroughs/programming-syntax.md

```diff
@@ -72,7 +72,7 @@ The branch name can be anything you like, for example:
 Please enter a name for your Instant Answer:
 ```
 
-Type `regex1` (since *regex* already exists in the repository, we'll add a character for this tutorial). The tool will do the rest:
+Type `regex1` (since *regex* already exists in the repository, we'll add a character for this tutorial). If the name contains more than one words, separate them with a space. The tool will do the rest:
```

Comment:
There's no need to send a new PR. This is the easiest way:

1. In the command line, make sure you're still in the same branch. I think you've been using "master". You can check with `git status`.
2. Make the edit to the file and save.
3. Commit the edit, e.g. `git commit -am "Small grammar fix"
4. Push the commit to your branch and repository on GitHub (also called "origin"), e.g. `git push origin master`.

This will update the pull request automatically.